### PR TITLE
Update add-data-translations.md

### PR DESCRIPTION
**Pull request by do-car**
I think it is worth mentioning how the main-entity definition class should be structured, in order to be able to get translated.

### CODE REVIEW

File: guides/plugins/plugins/framework/data-handling/add-data-translations.md [hunk 1]
```diff
@@ -274,3 +274,51 @@ class ExampleTranslationCollection extends EntityCollection
 ```
 {% endcode %}
 
+### Main Entity Class
+
+The main entity class, that is the class with the field(s) we are going to translate, must define:
+* a TranslatedField for the “name” field
+* a TranslationsAssociationField, with a reference to the ExampleTranslationDefinition
```
Comment (PaddyS) [inline guides/plugins/plugins/framework/data-handling/add-data-translations.md:281]:
```suggestion
* a `TranslatedField` for the “name” field
* a `TranslationsAssociationField`, with a reference to the ExampleTranslationDefinition
```

File: guides/plugins/plugins/framework/data-handling/add-data-translations.md [hunk 2]
```diff
@@ -274,3 +274,51 @@ class ExampleTranslationCollection extends EntityCollection
...
+
+{% code title="/src/Core/Content/Example/ExampleDefinition.php" %}
```
Comment (PaddyS) [inline guides/plugins/plugins/framework/data-handling/add-data-translations.md:283]:
```suggestion
{% code title="<plugin root>/src/Core/Content/Example/ExampleDefinition.php" %}
```

File: guides/plugins/plugins/framework/data-handling/add-data-translations.md [hunk 3]
```diff
@@ -274,3 +274,51 @@ class ExampleTranslationCollection extends EntityCollection
...
+
+    public function getCollectionClass(): string
+    {
+        return ExampleCollection::class;
+    }
+
+    public function getEntityClass(): string
+    {
+        return ExampleEntity::class;
+    }
```
Comment (PaddyS) [inline guides/plugins/plugins/framework/data-handling/add-data-translations.md:308]:
```suggestion
    [...]
```

**Review by PaddyS — CHANGES_REQUESTED** [review]

**Comment by PaddyS** [comment]
Whoa, thank you for your effort, @do-car! 🎉 
Just some minor changes that I'd like, besides that: Awesome!

**Comment by PaddyS** [comment]
Hey @do-car,

just a friendly reminder. Would be awesome if you could apply the suggested changes! :-)

**Comment by do-car** [comment]
> Hey @do-car,
> 
> just a friendly reminder. Would be awesome if you could apply the suggested changes! :-)
Hi @PaddyS, sorry for the wait :)

**Comment by PaddyS** [comment]
Awesome, thank you @do-car! 🎉

**Pull request merged**